test/unit/adapters/http.js

```diff
@@ -368,8 +368,8 @@ describe('supports http with nodejs', function () {
     }).listen(4444, function () {
       axios.get('http://localhost:4444/', {
         maxRedirects: 3,
-        beforeRedirect: function (options) {
-          if (options.path === '/foo') {
+        beforeRedirect: function (options, responseDetails) {
+          if ([options.path, responseDetails.headers.location].every(item => item === '/foo')) {
```

Comment:
Maybe it's better to use two simple conditions?

Reply:
Done ✅